CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# v4.1.0 (2024-12-10)
+
+## OS Changes
+* Enable plugins and detailed EBS volume stats for `nvme-cli` ([#269])
+* Set `LoaderTimeInitUSec` and `LoaderTimeExecUSec` in GRUB ([#273])
+* Update `ecs-agent` to v1.88.0 ([#303])
+* Third party package updates ([#308], [#311])
```

Comment:
Technically, the `ecs-agent` update counts as part of the third-party package updates.